macos/Sources/App/macOS/AppDelegate.swift

```diff
@@ -540,6 +546,9 @@ class AppDelegate: NSObject,
 
         case .always:
             NSApp.setActivationPolicy(.accessory)
+
+        case .quick_terminal:
+            NSApp.setActivationPolicy(NSApp.visibleRegularWindows.isEmpty ? .accessory : .regular)
```

Comment:
I think its cleaner to make something like a `syncActivationPolicy` function and call that from here and the `applicationDidUpdate` callback.